test_project/test_project/settings.py

```diff
@@ -106,6 +106,25 @@
 
 USE_TZ = True
 
+# Webmail datetime formats according to
+# https://docs.djangoproject.com/en/1.10/ref/templates/builtins/
+# modoboa-webmail is able to show the datetime of emails
+# in two ways.
+# If none is provided, default is RFC 5322 formatted date.
+# For this, USE_L10N must be set to True!
+
+# emails of the last week
+# en-us style
+WEBMAIL_SHORT_DATETIME_FORMAT = "l, P"
```

Comment:
I know part of the code used by the webmail might not be at the right place but the settings related to the webmail should be move the webmail repository. You can take example on the amavis plugin: https://github.com/modoboa/modoboa-amavis/blob/master/modoboa_amavis/settings.py.